tests/integration_tests/databases/api_tests.py

```diff
@@ -35,6 +35,8 @@
 
```

Comment:
Can we add a test on `SSHTunnel` failure, to make sure we don't save the `Database` object as wel..